theme-settings.php

```diff
@@ -150,6 +136,57 @@ function megatron_form_system_theme_settings_alter(&$form, &$form_state) {
 );
 */
 
+  /** CLF NAVIGATION OPTIONS
+  ---------------------------------------------------------- */
+  $form['clf_navigation_option'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Navigation Option'),
+    '#prefix' => '<div class="clf_navigation_option">',
+    '#suffix' => '</div>',
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+    '#weight' => -7,
+  );
+
+  $form['clf_navigation_option']['clf_navigation_placement'] = array(
+    '#type' => 'select',
+    '#title' => t('Choose the type of primary navigation that should be used on this website.'),
+    '#default_value' => theme_get_setting('clf_navigation_placement'),
+    '#options' => array(
+      'default' => t('Default CLF - Horizontal'),
+      'drawer--push-left' => t('Left Push Drawer'),
```

Comment:
Maybe to make the labels clearer 

```
Default: no drawer region
Drawer Region: push from left
Drawer Region: cover from left
Drawer Region: push from right
Drawer Region: cover from right
```

And I guess the `clf_navigation_placement` setting is poorly named, since it's not just for navigation. Maybe something more like `clf_drawer_region`?